content/blog/2017/2017-04-10-welcome-to-blue-ocean-pipeline-activity.adoc

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean's Pipeline Activity View"
```

Comment:
I think you should just refer to it as the "Activity View", since you can see freestyle jobs more or less the same. 

Also it's more compact :smile_cat: